samples/QmlArExample/qml/main.qml

```diff
@@ -39,7 +41,44 @@ ApplicationWindow {
         id: sceneView
         anchors.fill: parent
         scene: sceneLoader.item ? sceneLoader.item.scene : null
-        onMousePressed: arcGISArView.setInitialTransformation(mouse.x, mouse.y); // for touch screen events
+
+        // If m_screenToLocationMode is true, create and place a 3D sphere in the scene. Otherwise set the
+        // initial transformation based on the screen position.
+        onMousePressed: {
+            // If "screenToLocation" mode is enabled.
+            if (screenToLocationMode)
+            {
+                // Get the real world location for screen point from AR view.
+                const point = arcGISArView.screenToLocation(mouse.x, mouse.y);
+                if (!point)
+                    return;
+
+                // Get or create graphic overlay
+                var graphicsOverlay = graphicsOverlays.get(0);
+                if (!graphicsOverlay)
```

Comment:
I think this code would be cleaner if you check the graphicsOverlays size/count first before calling `.get(0)`.

This same comment applies to the C++ version.